memcachepp/memcache/policies/ketama_hash.hpp

```diff
@@ -0,0 +1,125 @@
+
+// Copyright 2015 (c) Bill <billtian945@gmail.com>
+// Distributed under the Boost Software License, Version 1.0.
+// (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
```

Comment:
I think you also might need to note that the original implementation of Ketama Hash may be licensed under BSD (or provide clear lineage of where Ketama hash may have been implemented first).